bedrock/settings/base.py

```diff
@@ -1125,6 +1126,13 @@ def get_default_gateway_linux():
         integrations=[DjangoIntegration()],
     )
 
+# Frontend uses the same DSN as backend by default, but we'll
+# specify a separate one for FE use in Production only
+SENTRY_FRONTEND_DSN = config(
+    "SENTRY_FRONTEND_DSN",
+    default=SENTRY_DSN,
+)
+
```

Comment:
Sentry's JS integration is already being called in `sentry.es6.js` as `Sentry.init()` so I don't think we need any more - it's just providing a different DSN URI to use